BinPAC.cmake

```diff
@@ -15,6 +15,12 @@ macro(BINPAC_TARGET pacFile)
         endif ()
 
         set(binpacDep "${BinPAC_EXE}")
+
+        # Ensure that for plugins included via --include-plugins, the Zeek's
+        # source tree paths are added to binpac's include path as well.
+        if ( NOT CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR )
```

Comment:
Could these actually be the same for `ZEEK_PLUGIN_INTERNAL_BUILD `? (Just asking, not sure)

Reply:
Yes, you're right. Even actual internal plugins (e.g. smb) will have a different `CMAKE_CURRENT_SOURCE_DIR`, so will just make it unconditional.